packages/convertkit/src/api/subscribe.ts

```diff
@@ -50,7 +50,7 @@ const subscribe = async (req: NextApiRequest, res: NextApiResponse) => {
 
       res.setHeader('Set-Cookie', convertkitCookie)
 
-      res.status(200).json(subscriber)
+      res.status(200).json({...subscriber, email: email_address, first_name})
```

Comment:
we have the subscriber (jsut an id), their submitted email address, and first name so we can return this information